.gitignore

```diff
@@ -1,3 +1,60 @@
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided this notice is
+# preserved.  This file is offered as-is, without any warranty.
+# Names of contributors must not be used to endorse or promote products
```

Comment:
I don't get your point about english. It is a mix of the GNU All-Permissive License and the 3-clause BSD license. https://www.gnu.org/prep/maintain/html_node/License-Notices-for-Other-Files.html https://directory.fsf.org/wiki/License:BSD_3Clause